packages/docs/pages/integrating-with-namada/interface.mdx

```diff
@@ -1,34 +1,35 @@
-   The CometBFT config will be located in `$BASE_DIR/$CHAIN_ID/cometbft/config/config.toml`. You will need to change the `cors_allowed_origins` field to `["*"]`. You can do this by running
+1. Follow the instructions for the [testnets](../networks/testnets.mdx) to connect to a testnet or [set up a local node](../networks/starting-network/local-network.mdx).
+2. Figure out where the base directory is stored and save its location in an environment variable with something like `export BASE_DIR=<path/to/base/dir>`.
+   You can follow [these docs](../networks/testnets/migrating-testnets.mdx) to save this variable. Consider saving the chain ID as an environment variable as well.
+   You can find the chain ID by running `cat $BASE_DIR/global-config.toml`. Save this chain-id to the environment variable `export CHAIN_ID=<CHAIN_ID>`.
+3. You will need to edit the CometBFT config in order to allow Namadillo to connect to your node.
+   The CometBFT config will be located in `$BASE_DIR/$CHAIN_ID//config.toml`. You will need to change the `cors_allowed_origins` field to `["*"]`. You can do this by running
    ```shell copy
    sed -i 's/cors_allowed_origins = \[\]/cors_allowed_origins = ["*"]/' \
-           $BASE_DIR/$CHAIN_ID/cometbft/config/config.toml
+           $BASE_DIR/$CHAIN_ID/config.toml
    ```
 
-#### Setting up the extension
+#### Setting up the Namada Keychain extension
 1. Clone the [namada-interface repository](https://github.com/anoma/namada-interface).
 2. Follow the installation instructions in `README.md`.
-3. `cd` into the `namada-interface/apps/extension` directory and run `yarn start:chrome`. This will build the extension and place it in the `namada-interface/apps/extension/build` directory. It also starts the dev server which will watch for changes.
-4. `cd` into the `namada-interface/apps/namada-interface` directory and run `yarn dev:local` in order to launch a local instance of the web wallet.
+3. `cd` into the `namada-interface/apps/extension` directory and run `yarn start:chrome`. This will build the Namada Keychain extension and place it in the `namada-interface/apps/extension/build` directory. It also starts the dev server which will watch for changes.
+4. `cd` into the `namada-interface/apps/namadillo` directory and run `yarn dev:local` in order to launch a local instance of Namadillo.
 4. Add the extension to the browser. For example, in Chrome, you can go to `chrome://extensions/` and click `Load unpacked` and select the `namada-interface/apps/extension/build/chrome/` folder.
 
 ## Receiving tokens
-You can show the address of any account by pressing the `Receive` button in the initial view under the "Total Balances" tab. You can copy the address by clicking the copy icon next to the address.
+You can show the address of any account by pressing the `Receive` button in the initial view under the `Total Balances` tab. You can copy the address by clicking the copy icon next to the address.
```

Comment:
This is no longer the case, there is no `Total Balances` tab:

```
You can show the address of the current account by hovering over the account name in the top right corner. Clicking it will copy it to the clipboard.
```